manuscript/book.md

```diff
@@ -2316,11 +2315,11 @@ For example, after reading the [The Programmers’ Stone](https://www.datapacrat
 ```js
 if (food === 'pizza')
 {
-	alert('Pizza ;-)');
+    alert('Pizza ;-)');
```

Comment:
```suggestion
  alert('Pizza ;-)');
```

Reply:
Hmm.. Might have to look at the formatter settings I use